config-linux.md

```diff
@@ -228,6 +186,46 @@ You can configure a container's cgroups via the `resources` field of the Linux c
 Do not specify `resources` unless limits have to be updated.
 For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
 
+#### Device whitelist
```

Comment:
On Wed, Jan 27, 2016 at 12:13:03PM -0800, Mrunal Patel wrote:
> > +#### Device whitelist
> 
> Should we call it an access list instead of a whitelist?

I'm just echoing the kernel docs [1] (which moved! [2]).  I've pushed
a855883 → ddd56d8 updating the link target.

[1]: https://www.kernel.org/doc/Documentation/cgroup-v1/devices.txt
[2]: https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=34a9304a96d6351c2d35dcdc9293258378fc0bd8